roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
+  command: "{{ 'keystone'|ursula_package_path(openstack_package_version) }}/bin/pip install {{ item }}"
+  register: pipout
+  changed_when: pipout.stdout|search("Successfully installed")
+  notify: restart keystone services
+  when: openstack_install_method == 'package' and
+        openldap.enabled|bool
+  with_items: keystone.package.ldap_python
```

Comment:
This whole task should be taken out. For LDAP testing, deploy keystone via source and install the extra bit that way. For package testing, we'll get the change made in giftwrap and indicate what package version has the necessary deps in the venv.